stdlib/tarfile.pyi

```diff
@@ -571,7 +571,7 @@ class TarInfo:
         self,
         *,
         name: str = ...,
-        mtime: int = ...,
+        mtime: int | float = ...,
```

Comment:
`int` is redundant in argument types, which is why CI fails:

```suggestion
        mtime: float = ...,
```

Reply:
Done, thanks.